src/packages/@ncigdc/utils/wrapSvg.js

```diff
@@ -171,7 +171,10 @@ export const wrapSvg: TWrapSvg = ({
     margins.top,
     beforeObject.height,
   ])})">
-        ${svg.innerHTML}
+        ${svg.innerHTML.replace(
+          /url\((?:['|"]*)(?:http|https):\/\/(?:.+)(#.+)(?:['|"]*)\)/g,
```

Comment:
that regex seems a little complicated, any reason it can't just be `/url\(['"]?https?:\/\/[^#]+(#.+)['"]?\)/g`?